data/spells/lib/spells.lua

```diff
@@ -312,38 +316,39 @@ function Player:addPartyCondition(combat, variant, condition, baseMana)
 	local members = party:getMembers()
 	members[#members + 1] = party:getLeader()
 
-	local position = self:getPosition()
 	local affectedMembers = {}
 	for _, member in ipairs(members) do
-		if member:getPosition():getDistance(position) <= 36 then
-			affectedMembers[#affectedMembers + 1] = member
+		local memberPosition = member:getPosition()
+
```

Comment:
Barely. At least be consistent: if you add a line here, add a line before the for above too.